Examples/WildeGuess/WildeGuess/WildeGuessCollectionViewController.mm

```diff
@@ -68,7 +68,7 @@ - (void)viewDidLoad
   [[[CKTransactionalComponentDataSourceChangesetBuilder transactionalComponentDataSourceChangeset]
     withInsertedSections:[NSIndexSet indexSetWithIndex:0]]
    build];
-  [_dataSource applyChangeset:initialChangeset mode:CKUpdateModeAsynchronous userInfo:nil];
+  [_dataSource applyChangeset:initialChangeset mode:CKUpdateModeSynchronous userInfo:nil];
```

Comment:
Thinking about this a bit more I don't think we can reliably verify changesets like this. For correctness it may be necessary to "fold" all pending changesets into the current state and verify the resulting state.